website/src/controllers/api-v3/auth.js

```diff
@@ -2,6 +2,8 @@ import validator from 'validator';
 import moment from 'moment';
 import passport from 'passport';
 import nconf from 'nconf';
+import setupNconf from '../../libs/api-v3/setupNconf';
+setupNconf();
```

Comment:
these calls shouldn't be necessary as long as the whole server is loaded. If you're requiring the controllers separately then setupNconf should be called from the tests